emote/memory/column.py

```diff
@@ -18,6 +20,31 @@ class Column:
     shape: Tuple[int]
     dtype: type
 
+    def configuration(self):
+        dtype_name = ""
+        if hasattr(self.dtype, "name"):
+            dtype_name = self.dtype.name
+        else:
+            dtype_name = self.dtype.__name__
+
+        return {
+            "shape": self.shape,
+            "dtype": dtype_name,
+        }
+
+    def configure(self, config):
```

Comment:
Why not name these two `state` and `load_state`?

Reply:
Because it's not state, just configuration. :-)

Reply:
I think it makes me a bit sad to follow exactly the same pattern we do for every single pytorch layer and emote callback but use different names for things.

Reply:
Fair point, fixed.